urbanopt/geojson-modelica-translator
# Cli bugfixing

#### Any background context you want to provide?
A Windows user reported problems with the CLI in the attached issue. This fixes that concern.
#### What does this PR accomplish?
- Removes explicit casting to a float
- Adds a log entry
- Updates dependencies
- Makes mos weatherfile variable name more explicit
- Updates geojson file used in test to match the SDK
#### How should this be manually tested?
I'd like a human Windows user (not just our Windows CI) to confirm this passes:
`poetry run pytest tests/management/test_uo_des.py::CLIIntegrationTest::test_cli_builds_sys_params`
#### What are the relevant tickets?
Resolves #503
